sefaria/model/marked_up_text_chunk.py

```diff
@@ -1,6 +1,10 @@
 from . import abstract as abst
 from sefaria.model.text import TextChunk, Ref
 from sefaria.system.exceptions import InputError, DuplicateRecordError
+from typing import List, Dict
```

Comment:
we no longer need to do these imports now that we're on Python 3.9. We can just write `list[str]` instead of `List[str]` and this doesn't require an import.